src/porepy/models/incompressible_flow_model.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, params: Optional[Dict] = None) -> None:
         self.variable: str = "p"
         self.mortar_variable: str = "mortar_" + self.variable
         self.parameter_key: str = "flow"
+        self._use_ad = True
```

Comment:
Is this included for consistency only? Just for curiosity.

Reply:
What do you mean?

Reply:
The module is Ad only, so no Assembler, I suppose? That is of course how it should be, and it probably is okay to have the attribute to be consistent with the older models.

Reply:
Yes, I think it is needed for now for safe inheritance from AbstractModel